the-content-guide/readme.md

```diff
@@ -1,20 +1,129 @@
 
 <img src="/images/contentcat.jpg" width="150">
 
-## File storage
+## Intro
 
-Each chapter should be it's own file. When you add a chapter, please provide the name of the file with a number. This number represents the order of the document chapters. After the number comes the chapters' name.
+Why do you need content marketing or branded content? Content marketing is a way to create a bond between a reader and a company. By creating content the reader will thoroughly enjoy, the reader will associate the brand with an awesome feeling and automatically think about that company first whenever searching for a product/service in that domain.
 
-## Pull requests
+Example: A solar cell company creates blogposts about:
+* how a solar cell works  
+* a comparison between green energy products
+* where is the future of solar cell power heading?
+* a photo collage of employees risking their lives on roofs
 
-Make your own branch. When working on the document, create a pull request for each document.
+So how do you attract readers and keep them engaged? There is an increasing demand for *'high quality content'*. But what is high quality content? What ingredients make a nice meal that people will remember? Better yet, how do you get people to ask for and share the recipe?
 
-## User story:
+This guide will describe what ingredients work well together and how you can best present the meal. It will allow you to attract your target audience and keep them maximally engaged. The guide will also explain how you can test your business metrics and how to reach your audience. Most of all, this guide will tell you how to create content you can be proud of!
 
-To know what makes good content — As a communication professional — I can read a comprehensive guide about content creation.
+## Originality & authenticity
+
+Good content is original. [Content Farms](https://en.wikipedia.org/wiki/Content_farm) are no longer wanted in these times. Search engines, such as Google, are getting better and better at detecting these 'clickbait sites' and these sites are not received well. Original content is sure to be shared more often. Moreover, why would you say something when it's not worth saying?
+
+> You share content because you want to achieve a certain goal. You want it to be read, and you want the reader to learn something from it.
+
+Apart from originality, high quality content also makes a certain promise about the credibility of the text. When sources have been used they should provide authority. Good sources are crucial and make sure that a text is taken seriously.
+
+Be a guardian of the originality and authenticity of your content piece. It's easier said than done while we all scrape information from different kinds of sources. When it's authority in people you interview, articles, books or your own judgement, it's all based on other work. The difference where you as an author come in, is how you handle these sources. Combine them to share new insights.
```

Comment:
Maybe you missed this re: line 32 onward: "While we all scrape information...". "While" indicates that "It's easier said then done" is happening during the process of scraping information. "When" or "because" would be more accurate because they are words that put these two clauses in some kind of conceptual chronology rather indicating that they happen at the same time. Make sense?

Also this: "When it's authority in people you interview...". I think this sentence could better begin with "If" because you're listing the source possibilities in the sentence. Also, one does not have authority in people, people have authority, so suggestion: "If it's the authority of people you interview...".

And this: "The difference where you as an author come in, is how you handle these sources". This just doesn't flow as a sentence. It sounds translated :) Flow and sentence order can be improved by removing the "where you come in" bit and removing the comma--before and after the comma are not separate clauses. Suggestion: "The difference is how you handle these sources as an author."